core/src/main/scala/filodb.core/store/IngestionConfig.scala

```diff
@@ -65,7 +68,8 @@ final case class StoreConfig(flushInterval: FiniteDuration,
                                "max-data-per-shard-query" -> maxDataPerShardQuery,
                                "evicted-pk-bloom-filter-capacity" -> evictedPkBfCapacity,
                                "ensure-headroom-percent" -> ensureHeadroomPercent,
-                               "metering-enabled" -> meteringEnabled).asJava)
+                               "metering-enabled" -> meteringEnabled,
+                               "resolution" -> resolution).asJava)
```

Comment:
Please add an example and docs in timeseries-dev-source.conf

Reply:
ack, added an example